pages/documentation/getting-started/fundamentals.md

```diff
@@ -92,7 +92,7 @@ Install USWDS from the command line, and save it as a dependency in your `packag
 npm install uswds --save
 
 # [a lot of notices]
-# + uswds@2.12.1 [or another version number]
+# + uswds@2.12.2 [or another version number]
```

Comment:
```suggestion
# + uswds@{{ site.uswds_version }} [or another version number]
```

Will give us the value from `uswds_version` in `_config.yml:5`. This way we don't have to keep manually updating.